apps/expo-go/android/expoview/src/main/java/host/exp/exponent/kernel/Kernel.kt

```diff
@@ -343,18 +343,9 @@ class Kernel : KernelInterface() {
     }
   private val kernelLaunchOptions: Bundle
     get() {
-      val exponentProps = JSONObject()
-      val referrer = exponentSharedPreferences.getString(ExponentSharedPreferences.ExponentSharedPreferencesKey.REFERRER_KEY)
-      if (referrer != null) {
-        try {
-          exponentProps.put("referrer", referrer)
-        } catch (e: JSONException) {
-          EXL.e(TAG, e)
-        }
-      }
       val bundle = Bundle()
```

Comment:
the bundle is just empty bundle now, so we may able to refactor this code a little bit. e.g. `private val kernelLaunchOptions = Bundle()`